en/contribute/docs.md

```diff
@@ -0,0 +1,158 @@
+# Contributing to Documentation
+
+Contributions to the DroneCore guides, including the PX developer and user guides are very welcome!
+This article explains how you can make changes, add content, and create translations.
+
+> **Note** You will need a (free) [Github](http://github.com) account to contribute to the guide.
+
+## Quick Changes
+
+Fixing typos or editing an *existing page* is easy:
+1. Click the **Edit** toolbar icon at the top of the relevant page in the guide.
+
+   ![Gitbook: Edit Page button](../../assets/gitbook/gitbook_toolbar_icon_edit.png)
+   
+   This will open the page for editing (in Github).
+1. Make the desired change.
+1. At the bottom of the page you'll be prompted to create a separate branch and then 
+   guided to submit a *pull request*.
+   
+The documentation team reviews submitted pull requests and will either merge
+it or work with you to update it.
+
+## Adding New Content - Big Changes
+
+### What Goes Where?
```

Comment:
@LorenzMeier This addresses your question "could we get these instructions into the dev guide so we can point people who want to write articles in the future to it?"
- from https://github.com/PX4/Devguide/pull/348#issuecomment-344853767